include/roboteam_ai/treeinterp/BTFactory.h

```diff
@@ -33,6 +33,8 @@ class BTFactory {
 
         static std::map<std::string, bt::BehaviorTree::Ptr> keeperRepo;
 
+        static std::map<std::string, bt::BehaviorTree::Ptr> codeTrees;
```

Comment:
It would be nice if public variables like these have a comment which explains the meaning of it.